fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/api/rdf/HttpResourceConverter.java

```diff
@@ -96,8 +97,7 @@ protected FedoraResource doForward(final Resource resource) {
 
     @Override
     protected Resource doBackward(final FedoraResource resource) {
-            //@TODO Implement this
-            return null;
+        return ResourceFactory.createResource(resource.getId());
```

Comment:
This class has been replaced by the `HttpIdentifierConverter` perhaps adding a `@deprecated` to this is now necessary?

Reply:
will add @Deprecated